frontend/components/sidebar/left/SidebarLeftFooter.vue

```diff
@@ -55,28 +55,28 @@
             >
               <SidebarLeftSelector
                 iconURL="bi:calendar-check"
-                btnText="New event"
+                :btnText="$t('components.sidebar.left.footer.new-event')"
```

Comment:
Should be `components.sidebar.sidebar-left-footer.new-event` and similar changes are needed here :)

Reply:
I'll fix that